docs/api-docs/headless/channels.mdx

```diff
@@ -14,6 +14,11 @@ You can create both using the [Channels API](/docs/rest-management/channels).
 2. Pass that channel ID in a `POST` request to `/channels/{channel_id}/site` to create a site for the provided channel.
```

Comment:
We also need to add additional steps here, that I've mentioned in the ticket.
Where developers need to:
1. Create site for headless channel which is going to be given channel primary domain.
2. Create checkout domain for given headless channel using https://developer.bigcommerce.com/docs/rest-management/channels/channel-site-checkout-url#upsert-a-site's-checkout-url
3. Add 3P SSL (optional) – https://developer.bigcommerce.com/docs/rest-management/sites/site-certificate#upsert-a-site%E2%80%99s-ssl/tls-certificate-information